bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCommandWorkItem.java

```diff
@@ -228,4 +228,22 @@ public String toString() {
     public String workItemName() {
         return "pr-command";
     }
+
+    /**
+     * This method returns all the comments in the pr including comments in reviews(review body) and file specific comments
```

Comment:
Given the discussion around only using "ordinary" PR comments, and not inline comments, I think the "and file specific comments" part is now wrong.

Reply:
> Given the discussion around only using "ordinary" PR comments, and not inline comments, I think the "and file specific comments" part is now wrong.

Good catch! Thanks Kevin. Will fix it.